trio/_core/tests/test_guest_mode.py

```diff
@@ -473,3 +474,24 @@ async def trio_main_raising(in_host):
     assert excinfo.value.__context__ is final_exc
 
     assert signal.getsignal(signal.SIGINT) is signal.default_int_handler
+
+
+def test_guest_mode_autojump_clock_threshold_changing():
+    # This is super obscure and probably no-one will ever notice, but
+    # technically mutating the MockClock.autojump_threshold from the host
+    # should wake up the guest, so let's test it.
+
+    clock = trio.testing.MockClock()
+
+    DURATION = 9999999
```

Comment:
Surely `10` would work just as well and would be less CI-hostile if it breaks :-)